activiti-spring-process-extensions/src/main/java/org/activiti/spring/process/ProcessVariablesInitiator.java

```diff
@@ -45,21 +45,39 @@ public ProcessVariablesInitiator(ProcessExtensionService processExtensionService
         this.variableValidationService = variableValidationService;
     }
 
-    @Override
-    public ProcessInstance createAndStartProcessInstanceWithInitialFlowElement(ProcessDefinition processDefinition, String businessKey, String processInstanceName, FlowElement initialFlowElement, Process process, Map<String, Object> variables, Map<String, Object> transientVariables, boolean startProcessInstance) {
+    public Map<String, Object> getVariablesFromExtensionFile(ProcessDefinition processDefinition,
```

Comment:
Can we rename it to `calculateVariablesFromExtensionFile`?

Reply:
ok